package.json

```diff
@@ -1,74 +1,73 @@
 {
-    "name": "heliostats",
```

Comment:
Note, the changes are literally just indentation from what I can see. But we want this file to exactly match [package.json in the `heliostats` repo](https://bitbucket.org/kwh/heliostats/src/develop/package.json).